File: beginner/java/1009.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

            scanner.useLocale(Locale.US);

            String nome = scanner.nextLine();
            double salarioFixo = scanner.nextDouble();
            double totalVendas = scanner.nextDouble();

            double valorGanho = salarioFixo + (totalVendas * 0.15);

            System.out.printf("TOTAL = R$ %.2f%n", valorGanho);

        scanner.close();
    }
}

File: beginner/c/1009.c
#include <stdio.h>

int main(){

    char nome[255];
    double salario, totalVendas;

    scanf("%s", nome);
    scanf("%lf", &salario);
    scanf("%lf", &totalVendas);

    double total = salario + (totalVendas * 0.15);
    
    printf("TOTAL = R$ %.2lf\n", total);

    return 0;
}
